docs/css/custom.css

```diff
@@ -143,6 +143,12 @@ a {
 	color:black;
 }
 
+#subscribe a {
+	color: #824d67;
+	font-family: system-ui;
```

Comment:
Waarom wil je hier system-ui gebruiken? Dan wordt het font van het os gebruikt, afhankelijk van taal instellingen toch? Dat zou dan voor de ene gebruiker een andere ervaring opleveren dan de andere. Op zich wel grappig maar dan zou ik dat voor de hele site doen. Dat scheelt vervolgens ook een font download, maar dat doen we al teniet elders op de site.

Reply:
I replaced this font with `sans-serif` which looks similar to it and is less dynamic ;-)